docs/firebase.md

```diff
@@ -277,8 +280,8 @@ app:
             .querySelector('#signinBtn')
             .addEventListener('click', signIn);
           document
-            .querySelector('#tellJoke')
-            .addEventListener('click', generateJoke);
+            .querySelector('#suggestMenuItem')
+            .addEventListener('click', generateMenuItem);
 
```

Comment:
I believe there is a webapp that goes along with all of this, hence why all of the dom related methods here. Are we updating that as well?